configure.ac

```diff
@@ -290,28 +290,15 @@ AS_IF([test "$want_pi" = yes], [
 	AS_IF([test "x$GRPC_CPP_PLUGIN" = x], [
 		AC_MSG_ERROR([grpc_cpp_plugin not found])
 	])
+])
 
-	AS_IF([test "$PYTHON" != :], [
-		AC_PATH_PROG([GRPC_PY_PLUGIN], [grpc_python_plugin])
-		AS_IF([test "x$GRPC_PY_PLUGIN" = x], [
-			AC_MSG_WARN([grpc_python_plugin not found, Python code won't be generated])
-		])
-	])
-	AM_CONDITIONAL([HAVE_GRPC_PY_PLUGIN], [test "x$GRPC_PY_PLUGIN" != x])
-
-	AC_PATH_PROG([GRPC_CPP_PLUGIN], [grpc_cpp_plugin])
-	AS_IF([test "x$GRPC_CPP_PLUGIN" = x], [
-		AC_MSG_ERROR([grpc_cpp_plugin not found])
-	])
-
-	AS_IF([test "$PYTHON" != :], [
-		AC_PATH_PROG([GRPC_PY_PLUGIN], [grpc_python_plugin])
-		AS_IF([test "x$GRPC_PY_PLUGIN" = x], [
-			AC_MSG_WARN([grpc_python_plugin not found, Python code won't be generated])
-		])
-	])
-	AM_CONDITIONAL([HAVE_GRPC_PY_PLUGIN], [test "x$GRPC_PY_PLUGIN" != x])
+AS_IF([test "$PYTHON" != :], [
+    AC_PATH_PROG([GRPC_PY_PLUGIN], [grpc_python_plugin])
+    AS_IF([test "x$GRPC_PY_PLUGIN" = x], [
+        AC_MSG_WARN([grpc_python_plugin not found, Python code won't be generated])
+    ])
 ])
```

Comment:
I don't think it makes sense not to condition this on `test "$want_pi" = yes`.

Reply:
When the the condition is on `test "$want_pi" = yes`, `./autogen.sh && ./configure` fails with:

```
configure: error: conditional "HAVE_GRPC_PY_PLUGIN" was never defined.
Usually this means the macro was only invoked conditionally.
```